feat(ReleaseNote): Archiving Contract Data can now be disabled (#834)
bonitasoft/bonita-doc: md/configurable-archive.md

* To archive a receive task each time it changes state, set `bonita.tenant.archive.SReceiveTaskInstance` to true. This information is not available in Bonita Portal.
* To archive a data instance each time it changes value, set `bonita.tenant.archive.SDataInstance` to true. 
* To archive a reference between a business data and its process instance, set `bonita.tenant.archive.RefBusinessDataInstance` to true. 

> * To archive a reference to a contract data and its process instance, set `bonita.tenant.archive.SContractData` to true.  Note that, if not set, this parameter will by default be set to true. 

Reviewer: ```suggestion
* To archive contract data when a task / process instance finishes, set `bonita.tenant.archive.SContractData` to true.  Note that, if not set, this parameter will be set to true by default.
```
